src/reducers/RegisterReducer.js

```diff
@@ -0,0 +1,26 @@
+import { REGISTER_USER, REGISTER_FAIL } from '../actions/types';
+
+const initialState = {
+  registerSuccess: false,
+  error: null,
+};
+
+const RegisterReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case REGISTER_USER:
+      return {
+        ...state,
+        registeredSuccess: true,
+        register: action.payload,
+      };
+    case REGISTER_FAIL:
+      return {
+        ...state,
+        error: true,
```

Comment:
set registerSuccess to be false. Also, what component do we use registerSuccess for? I guess this should be used to update the register form to say register failed right? For now, we can just leave it out. We can work together tomorrow to implement some component for it once we're hooked up to the backend

Reply:
Register success should be hooked up to the /authenticate-email component I made

Reply:
https://github.com/acmucsd/membership-portal-ui/pull/96